.changeset/eleven-baboons-sparkle.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-react': minor
```

Comment:
Both changeset's should be `patch` not `minor` 👍

Reply:
Changed it 👍